File: Darli/app/src/main/java/com/example/darli/data/model/InfoPondok.kt
package com.example.darli.data.model

import java.io.Serializable

data class InfoPondok(
    val id: Int,
    val judul: String,
    val konten: String,
    val jenis: String,
    val gambar: String?,
    val created_at: String
) : Serializable

data class InfoPondokResponse(
    val response_code: Int,
    val content: List<InfoPondok>
)

File: AlumniDarli/app/Models/InfoPondok.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class InfoPondok extends Model
{
    use HasFactory;

    protected $table = 'info_pondok';

    protected $fillable = [
        'judul',
        'konten',
        'jenis',
        'gambar',
    ];

    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
    ];
}
